File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/midlandfcstresponse/MidLandFcstItems.java
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.midlandfcstresponse;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.tickaroo.tikxml.annotation.Element;
import com.tickaroo.tikxml.annotation.Xml;

import java.util.List;

@Xml(name = "items", inheritance = true)
public class MidLandFcstItems {
    @Expose
    @SerializedName("item")
    @Element(name = "item")
    private List<MidLandFcstItem> item;
    
    public void setItem(List<MidLandFcstItem> item) {
        this.item = item;
    }
    
    public List<MidLandFcstItem> getItem() {
        return item;
    }
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/kma/json/midlandfcstresponse/MidLandFcstItems.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.kma.json.midlandfcstresponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "items", inheritance = true)
class MidLandFcstItems {
    @Expose @SerializedName("item") @Element(name = "item") var item: List<MidLandFcstItem>? = null
}